Testreglar/252/Nett/nett-2.5.2a.json

```diff
@@ -0,0 +1,189 @@
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		},
+		{
+			"title": "3.7"
+		}
+	],
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "URL/Side-ID:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Finnes det funksjonalitet som kan betjenes med en enkel pekerbevegelse (single pointer)?",
+			"ht": "<p>Enkel pekerbevegelse er pekerinndata som fungerer med kun ett kontaktpunkt med skjermen, herunder:</p>\n<ul>\n<li>enkeltrykk og -klikk,</li>\n<li>dobbelttrykk og -klikk,</li>\n<li>lange trykk</li>\n</ul>\n<p>Slik identifiserer du funksjonalitet som kan betjenes med en enkel pekerbevegelse:</p>\n<ul>\n<li>Åpne testsiden på en enhet med berøringsskjerm f.eks. smarttelefon eller nettbrett. </li>\n<li>Slå av skjermleser og andre tilgjengelighetsfunksjoner, hvis det er aktivert på enheten\n<ul>\n<li>Dette gjelder både hjelpemiddelteknologi og innstillinger i operativsystemet</li>\n</ul>\n</li>\n<li>Gjør en visuell inspeksjon og sjekk om det finnes interaktive elementer, som lenker og knapper eller lignende funksjonalitet</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Det finnes ikke funksjonalitet som kan betjenes med en enkel pekerbevegelse."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Hvilken funksjonalitet tester du?",
+			"ht": "<p>Beskriv funksjonaliteten, slik at det er mulig å identifisere den i ettertid. Hvis det gjelder flere funksjoner, registrerer du en og en.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "Funksjonalitet:",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			},
+			"multilinje": true
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Utløser trykkhandlingen (down-event) med pekeren funksjonen?",
+			"ht": "<p>Slik tester du:</p>\n<ul>\n<li>Trykk og hold nede pekeren du bruker (museknappen, fingeren eller stylus) på innholdet på berøringsskjermen</li>\n<li>Sjekk om trykkhandlingen (down-event) utløser funksjonen\n<ul>\n<li>Eksempel: En lenke åpnes med en gang du trykker på den.</li>\n</ul>\n</li>\n</ul>\n<p>Merknad: Ikke hold nede pekeren lenge, fordi innholdsprodusent kan ha implementert innhold som kommer opp ved hold event.</p>",
```

Comment:
gjelder kravet for  nettsider som brukes på berøringsskjerm? Eller skulle det vært endret i første kulepunkt?

Reply:
Kravet gjelder både berøringsskjerm og enheter som støtter musepekeren. Vi tester på berøringsskjerm.